docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -19,11 +19,11 @@ cd /opt/coins/blockbook/bitcoin_regtest
 
 echo "Starting blockbook service"
 bin/blockbook \
-    -blockchaincfg=config/blockchaincfg.json \
+    -blockchaincfg=/opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json \
     -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db \
     -sync \
     -internal=:19021 \
     -public=:19121 \
-    -certfile=cert/blockbook \
     -explorer= \
-    -log_dir=logs
+    -log_dir=/opt/coins/blockbook/bitcoin_regtest/logs \
```

Comment:
@vdovhanych would be nice to have this log somehow easily accessible. But let's worry about that later..

Reply:
We could extract it as artifact in the gitlab pipeline in the same way we do with the tenv logs. Could be done later.

Reply:
Will you open an issue pls?